brd-ios/breadwallet/src/Redesign/Scenes/Swap/SwapMain/SwapMainCoordinator.swift

```diff
@@ -0,0 +1,22 @@
+// 
+//  SwapMainCoordinator.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import Foundation
+
+class SwapMainCoordinator: BaseCoordinator, SwapMainRoutes {
```

Comment:
remove Main ... SwapCoordinator is descriptive enough and main rly does not add anything